sdk/core/azure-core/tests/async_tests/test_rest_stream_responses_async.py

```diff
@@ -217,3 +216,32 @@ async def test_pass_kwarg_to_iter_raw(client):
     response = await client.send_request(request, stream=True)
     async for part in response.iter_raw(chunk_size=5):
         assert part
+
+@pytest.mark.asyncio
+async def test_decompress_compressed_header(client):
```

Comment:
As discussed in chat - lets make sure we duplicate these tests across the transports to ensure consistent behaviour.